.github/ISSUE_TEMPLATE.md

```diff
@@ -6,24 +6,32 @@
 
 ### Description
 
-[Description of the issue]
+Description of the issue and use the labels to add additional context.
 
 ### Steps to Reproduce
 
 1. [First Step]
 2. [Second Step]
 3. [and so on...]
 
-**Expected behavior:** [What you expect to happen]
+**Expected behavior:**
 
-**Actual behavior:** [What actually happens]
+What you expect to happen
 
-**Reproduces how often:** [What percentage of the time does it reproduce?]
+**Actual behavior:**
+
+What actually happens
+
+**Reproduces how often:**
+
+What percentage of the time does it reproduce?
 
 ### Versions
 
 The version is found in `version.txt`.
 
+If it is a dev version, include the result of `git rev-parse --short HEAD`.
```

Comment:
i'm not a bit fan of using git rev-parse because (i) it requires being in the repos and (ii) it is not intuitive (e.g. not clear what branch it is.

i would instead report the branch/SHA which is displayed at the beginning of each SCT command. Example: ```Spinal Cord Toolbox (move-assets/0215da7826e56b4d1066c9869cfcaa778fbc25ef)```